### ISSUE TITLE
Advanced Theme Structure

### ISSUE THREAD

**Issue by zbtirrell** [OctopodGames/Rodents-Revenge#8]
Thoughts on building a theme to drive the site:

1) come up with a name for the theme.  It should be all lowercase with no spaces or punctuation (ex. octopodgames-theme)

2) create repository matching the theme name

3) Get setup with access to a dev WordPress environment for testing - I'm happy to set this up for you. (ex. ryley.octopodgames.com)
  3-1) make sure you can login to the WP Dashboard
  3-2) make sure you can get files to and from where they are on the server (SFTP or SSH/vi most likely)

4) fork the repo on github into your account

5) git clone your copy of the repo into the dev site ( wp-content/themes/ )

6) Choose a theme to base our theme off of.  A possible option is integrating Bootstrap into twentytwelve (http://wordpress.org/extend/themes/twentytwelve) but there may be perfectly good themes that have already done that or similar, let's not spend too much time recreating the wheel.

7) put the theme we want to base stuff off in the dev site ( wp-content/themes/ )

8) create a style.css file in our theme that establishes a child theme relationship with the Bootstrap (or twentytwelve) parent. (See: http://codex.wordpress.org/Child_Themes)

9) Enable our theme.  It _should_ look just like the parent theme.  Celebrate.

10) create a games/ subfolder in the theme

11) check out the various game repositories as submodules in the games folder ( see: http://git-scm.com/book/en/Git-Tools-Submodules )

12) create a functions.php file that defines a shortcode to load each of the games.  This could be one shortcode with attributes or multiple shortcodes, unique for each game. (See: http://codex.wordpress.org/Shortcode_API)

Note: committing any successful progress along the way on this would be helpful for collaboration.  Ideally this should be in place before class on 7/13.

@rameden  - You are lead on this task.  I'm looking at you to get it all pulled together.  If there are subcomponents or steps you would like Catherine (or others) to do, please request assistance (perhaps via comments on this ticket for clarity and feedback)

@helloitsacat - You need to understand how all of this works because you will be living in this world extensively in the next steps of the project.  Definitely read the linked resources.  If you want to mirror Ryley's efforts, that also might be helpful for learning, though not required.  If there are pieces you can do for or in advance or Ryley, I'm sure he would be appreciative. :)

@zcdziura - this will effect the deploy process.  You should take the time to read the linked resources and understand (especially git submodules)

@Camwyn - This effects how the UI of everything is put together,  you should take the time to read the linked resources and understand.

**Comment by rameden** [comment]
ryley.octopodgames.com is using a child theme. Woo
